src/App.js

```diff
@@ -33,6 +33,10 @@ export const App = () => {
       return Boolean(essentialForAllPaths && supportPageData);
     }
 
+    if (path === "/schedule") {
+      return true;
+    }
```

Comment:
Not really a fan of my 'essentialDataFetchingFinished' function at all. Typescript + ReactQuery would help improve this loading behaviour massively, one for another time.

Reply:
When you were writing this, `useQuery` was probably only just an idea in Tanner's head ;)